### PR TITLE
Add the agent to the docker group if the docker check is enabled

### PR DESCRIPTION
**Pull request by hush-hush**
This is mandatory to monitor Docker.

Fix: #130

### CODE REVIEW

File: tasks/agent6.yml [hunk 1]
```diff
@@ -52,6 +52,26 @@
...
+    register: docker_group_exists
+  - name: Add the dd-agent user to the docker group
+    user:
+      name: dd-agent
+      groups: docker
+      append: yes
+    notify: restart datadog-agent
+    when: docker_group_exists.stdout|length > 0
+  # only try to add the agent to the docker group if the docker check is enabled
+  when: datadog_enabled and not ansible_check_mode and "docker" in datadog_checks
```
Comment (okunc) [inline tasks/agent6.yml:73]: the list of checks in datadog_checks is free [no naming conventions] so this check may not pass if someone customizes their docker check to e.g. docker_montitor or whatever..

File: tasks/agent6.yml [hunk 2]
```diff
@@ -52,6 +52,26 @@
...
+    # found. Using 'ignor_errors' still print the error message and disabling
+    # logs output for this task still logs a message in red saying that logs
+    # were discarded.
+    shell: "grep -E '^docker:' /etc/group | cat"
+    args:
+      warn: no
+    register: docker_group_exists
+  - name: Add the dd-agent user to the docker group
+    user:
+      name: dd-agent
```
Comment (okunc) [inline tasks/agent6.yml:67]:
```suggestion
      name: "{{ datadog_user }}"
```

File: tasks/agent6.yml (hunk 1 above)
Comment (hush-hush) [reply tasks/agent6.yml:73]: This support the official docker check which is always named `docker`, for custom checks users have to handle there setup on there own.

With the Agent the name of the configuration folder in `/etc/datadog/conf.d` determine the associated check for the configuration files inside.

**Comment by hush-hush** [comment]
Closing this PR for now until #167 is merged.

**Pull request closed without merging**